_posts/2022-02-17-sergei-tikhomirov.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Sergei Tikhomirov and Lightning privacy"
+tags:
+  - research
+  - Lightning
+---
+
+Postdoc Researcher [Sergei](https://twitter.com/serg_tikhomirov) joins Murch and Jonas to talk about channel balance probing in Lightning, privacy concerns in general, and the importance of researcher-developer collaboration.
+
+<!-- iframe and anchor link go here -->
```

Comment:
I don't think I can push a new commit without closing this PR so here is the anchor links markdown
```
<iframe src="https://anchor.fm/chaincode/embed/episodes/Sergei-Tikhomirov-and-Lightning-privacy-e1egh3e" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>

[anchor.fm](https://anchor.fm/chaincode/episodes/Sergei-Tikhomirov-and-Lightning-privacy-e1egh3e)
```